setup-demo.sh

```
@@ -0,0 +1,285 @@
+# Health Verification
+#######################################################################
+
+verify_health() {
+    log_step "🏥 Verifying Environment Health"
+
+    # Check kubectl context
+    log_info "Kubernetes cluster info:"
+    kubectl cluster-info --context "kind-${CLUSTER_NAME}" 2>/dev/null || kubectl cluster-info
+
+    # Check pod status summary
+    local total_pods=$(kubectl get pods --all-namespaces --no-headers 2>/dev/null | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)